src/app-gocardless/bank-factory.js

```diff
@@ -99,6 +99,9 @@ export const BANKS_WITH_LIMITED_HISTORY = [
   'SEB_EEUHEE2X',
   'SEB_UNLALV2X',
   'SELLA_PERSONAL_CREDIT_SELBIT22',
+  'SANTANDER_BSCHESMM',
+  'NORDEA_NDEADKKK',
```

Comment:
Mind removing these two lines? They are duplicates of above

Reply:
Sorry about that! I was careless with the merge. Done ✅

Reply:
All good!